.babelrc

```
@@ -0,0 +1,8 @@
+{
+  "presets": [
+    [ "@babel/preset-env", {
+      "targets": { "node": "10" }
```

iirc this works - wdyt?
```suggestion
      "targets": { "node": "current" }
```